tools/oeedger8r/Emitter.ml

```diff
@@ -683,6 +683,10 @@ let oe_gen_ocall_enclave_wrapper (os:out_channel) (uf:Ast.untrusted_func) =
   fprintf os "\n";
   fprintf os "{\n";
   fprintf os "    oe_result_t _result = OE_FAILURE;\n\n";
+  fprintf os "    /* If the enclave is in crashing/crashed status, new OCALL should fail\n";
```

Comment:
This logic is similar to the [old implementation](https://github.com/Microsoft/openenclave/blob/e504559e9ea03dfd25f8b2f58a96b12c9c140e48/enclave/core/sgx/calls.c#L548).